src/sentry/sentry_metrics/indexer/mock.py

```diff
@@ -18,25 +19,43 @@
     "staging": 10,
     "user": 11,
     "init": 12,
+    "session.error": 13,
 }
-_REVERSE = {v: k for k, v in _STRINGS.items()}
 
 
-class MockIndexer(StringIndexer):
-    """
-    Mock string indexer
-    """
+class SimpleIndexer(StringIndexer):
+
+    """Simple indexer with in-memory store. Do not use in production."""
+
+    def __init__(self):
+        self._counter = itertools.count(start=len(_STRINGS))
+        self._strings = DefaultDict(self._counter.__next__)
```

Comment:
As commented above I don't think `typings.DefaultDict` is supposed to be used like this, could we use `collections.defaultdict` instead ?